src/components/Lessons/lessonNavigator.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+    
+
+</script>
+
+<div class="navbar bg-base-100 border-b-2 border-secondary">
+    <div class="navbar-start w-max mr-8">
+      <div class="dropdown">
```

Comment:
What's the dropdown for?

Reply:
place holder right now, supposed to be the menu

Reply:
Which menu?